NEWS

```diff
@@ -1,22 +1,45 @@
-x/y/2022 ola-0.10.9
+x/y/2023 ola-0.10.9
  Features:
- *
+ * Python 3 compatibility across the board (including the RDM Responder Tests)!
```

Comment:
It's not yet solved but since 0.10.9 won't be released before this is fixed, I assume we can already mention it in the NEWS

Reply:
Yeah I must admit I didn't really think about this, I guess the responsible thing will be to take both of these lines out that you've flagged and put them in #1761 in case we did end up with a more ad-hoc Debian release...

Reply:
Reverted in https://github.com/OpenLightingProject/ola/pull/1761/commits/c41589de0b67015f5d94eafb061eba511801ef69